Disc/Networking/APIClient.swift

```diff
@@ -2,6 +2,7 @@ import Swish
 
 /// An API client for Passport.
 public struct APIClient {
+    static let auth = Swish.APIClient()
```

Comment:
This name is probably a bit misleading since it's really for making unauthorized requests.